lib/ddg-atbgrabber.js

```diff
@@ -30,6 +29,7 @@ exports.install = function () {
           // set the ATB param from the loaded site if it was not set yet
           if (atbParam && !ss.storage.atb_set) {
             ss.storage.atb = atbParam;
+            ss.storage.set_atb = atbParam;
             ss.storage.atb_set = true;
```

Comment:
I don't think we need both of these. With the Chrome extension we just re-used the existing boolean. I think we should probably do the same here unless there's a reason we can't?